.github/workflows/react_tests.yml

```
@@ -1,4 +1,4 @@
-name: React Tests (non-blocking)
+name: React Tests (blocking)
```

I'd go with just this, as Github already throws up a "Required" label if it's required.

```suggestion
name: React Tests
```